docs/contributors/readme.md

```diff
@@ -1,18 +1,33 @@
-# Contributor Documentation
+# Contributors Guide
```

Comment:
Should be "Contributor Guide" or "Contributor's Guide"

Reply:
I standardized on Contributor Guide, this matches the [Core Contributor Handbook](https://make.wordpress.org/core/handbook/) which is singular non-possessive.